kolibri_explore_plugin/collectionviews.py

```diff
@@ -565,12 +565,15 @@ def _build_remotechannelimport_task(channel_id):
 
 _content_manifests = []
 _content_manifests_by_grade_name = {}
-_collection_download_manager = CollectionDownloadManager()
+_collection_download_manager = None
 
 
-def _read_content_manifests():
+def _initiate():
     global _content_manifests
     global _content_manifests_by_grade_name
+    global _collection_download_manager
+
+    _collection_download_manager = CollectionDownloadManager()
```

Comment:
I'm not sure why the download manager initialization needs to be delayed. The only time the manifests come into play is when other code calls `_collection_download_manager.from_manifest()` or `_collection_download_manager.from_state()`.

Reply:
Well, today the manifests are read and parsed at import time. For example when `kolibri manage migrate` is called and this plugin is enabled. Not ideal. Worse would it be to add an online fetch when running migrations.